engine-rest/engine-rest-openapi/src/main/templates/paths/process-instance/{id}/suspended/put.ftl

```diff
@@ -16,7 +16,8 @@
 
   <@lib.requestBody
       mediaType = "application/json"
-      dto = "SingleProcessInstanceSuspensionStateDto" />
+      dto = "ProcessInstanceSuspensionStateDto"
+      requestDescription = "**Only allowed property**: `suspended`, the others will be ignored" />
```

Comment:
Why don't we use `SuspensionStateDto` here then? Maybe we can refactor this in the REST API implementation as well.